test/runtests.jl

```diff
@@ -146,36 +146,36 @@ mt6=MT6(5.4, 6) # outer positional
 @test_throws  TypeError MT6{String, Int}("asdf", 6)
 
 # user defined BAD inner positional constructor
-@with_kw type MT7{R,I<:Integer} <: AMT{R}
+@with_kw mutable struct MT7{R,I<:Integer} <: AMT{R}
     r::R=5
     a::I
-    @compat (::Type{MT7{R,R}}){R}(r::R) = new{R,R}(r,r+8)
+    (::Type{MT7{R,R}})(r::R) where {R} = new{R,R}(r,r+8)
```

Comment:
bad bot

Reply:
Note how this line did not get translated but the one here https://github.com/mauro3/Parameters.jl/pull/40/files#diff-fce720c43af3c52c862fd7451c7374b8R169 did. The only difference is that the translated one is in an expression.

There are several instances.